config/application.rb

```diff
@@ -12,6 +12,10 @@ class Application < Rails::Application
     config.load_defaults(7.1)
     config.autoloader = :zeitwerk
 
+    # Autoload subfolders of modules (recursively)
+    # https://stackoverflow.com/a/4794775/
+    config.autoload_paths += Rails.root.glob("app/models/**/")
+
```

Comment:
Unfortunately, as of d33072d, the ERD creation does not work anymore as it cannot find some classes. This is probably due to this modified `autload_paths`. Will investigate.

Reply:
Fixed in 4ed73bccabf9